Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptAddView.jsp

```diff
@@ -208,18 +249,51 @@
 		}
 		$("#synonymsids").val(synonyms);
 		$('#addedSynonnym').val(synonyms);
-
 	};
-
+	var bindingRemoval = function(row) {
+		var ids = $('#synonymId').val();
+		var idArray = ids.split(',,');
+		var newIdArray = '';
+		for (var i = 0; i < idArray.length; i++) {
+			if (i != row) {
+				console.log(i + 'i');
+				console.log(row + 'row');
+				newIdArray = newIdArray + idArray[i];
+				newIdArray = newIdArray + ',,';
+				console.log(newIdArray);
+			}
+		}
+		console.log(newIdArray);
+		$('#synonymId').val(newIdArray);
+		var synonymDescriptions = $('#synonymDescription').val();
+		var synonymDescriptionArray = synonymDescriptions.split(',,');
+		var newSynonymArray = '';
+		for (var i = 0; i < synonymDescriptionArray.length; i++) {
+			if (i != row) {
+				newSynonymArray = newSynonymArray + synonymDescriptionArray[i];
+				newSynonymArray = newSynonymArray + ',,';
+			}
+		}
+		$('#synonymDescription').val(newSynonymArray);
+		var terms = $('#term').val();
+		var termArray = terms.split(',,');
+		var newTermArray = '';
+		for (var i = 0; i < termArray.length; i++) {
+			if (i != row) {
+				newTermArray = newTermArray + termArray[i];
+				newTermArray = newTermArray + ',,';
```

Comment:
@jdamerow  As far as I remember, I added two commas as separator for terms because terms itself might contain **,** so made the separator as **,,**